api/lib/interfaces/controllers/security-controller.js

```diff
@@ -186,6 +187,22 @@ async function checkUserIsAdminInScoOrganizationAndManagesStudents(request, h) {
   return _replyWithAuthorizationError(h);
 }
 
+async function checkUserCanAccessCampaign(request, h) {
+  if (!request.auth.credentials || !request.auth.credentials.userId) {
+    return _replyWithAuthorizationError(h);
+  }
+
+  const userId = request.auth.credentials.userId;
+  const campaignId = parseInt(request.params.id);
+
+  const canAccessCampaign = await checkUserCanAccessCampaignUseCase.execute(userId, campaignId);
+  if (canAccessCampaign) {
+    return h.response(true);
+  }
```

Comment:
Si tu renommes le use-case en `userCanAccessCampaign`, tu peux le faire en une ligne (parce qu'il n'y a qu'un seul export)
```suggestion
  if (await userCanAccessCampaignUseCase(userId, campaignId);) {
    return h.response(true);
  }
```